apps/spruce/src/components/EditableTagField/TagRow.tsx

```diff
@@ -138,6 +138,7 @@ const FlexColumnContainer = styled.div`
   display: flex;
   flex-direction: column;
   margin-right: ${size.xs};
+  flex-grow: 1;
 `;
 
```

Comment:
let's change the button text from title case to sentence case: `Add Parameter` => `Add parameter`. This is based on the guidance of the design team